### ISSUE TITLE
IPL command without a LOADPARM

### ISSUE THREAD

**Issue by arfineman** [SDL-Hercules-390/hyperion#296]
When IPL command without LOADPARM is issued, it uses the LOADPARM from the previous command. It should not. It should use the one from the config file, if one exist. None if there is none. LOADPARM from the IPL command should not be carried over from one command to another.

**Comment by arfineman** [comment]
The last sentence should read 'LOADPARM should not be carried over from one command to another'.

**Comment by Fish-Git** [comment]
In other words, the loadparm option on the IPL command should be treated as only a _temporary override_ of whatever the _default_ loadparm is, with the default loadparm being: whatever was defined in the configuration file (i.e. via the loadparm command itself). Is that correct?

For example:

```
LOADPARM FOO        # from config file -OR- from "loadparm" panel command
IPL 0123
LOADPARM
```
should perform an IPL with the loadparm set to "FOO" and the "LOADPARM" command (without any arguments) should report that the _default_ loadparm is currently set to "FOO", whereas:

```
LOADPARM FOO
IPL 0123 LOADPARM BAR
LOADPARM
```
should perform an IPL with the loadparm set to "BAR" and the "LOADPARM" command (without any arguments) should report that the _default_ loadparm is _STILL_ currently set to "FOO", and finally:

```
IPL 0123
LOADPARM
```
should perform an IPL with the loadparm set to blanks (i.e. nothing) and the "LOADPARM" command (without any arguments) should report that the _default_ loadparm is currently blanks, and:

```
IPL 0123 LOADPARM BAR
LOADPARM
```
should perform an IPL with the loadparm set to "BAR" and the "LOADPARM" command (without any arguments) should report that the _default_ loadparm is still currently set to blanks, and:

```
IPL 0123 LOADPARM BAR
LOADPARM
LOADPARM FOO
LOADPARM
```
should perform an IPL with the loadparm set to "BAR" and the first loadparm command (without any arguments) should report that the _default_ loadparm is currently set to blanks, the second loadparm command should _SET_ the default loadparm to "FOO", and the third loadparm command (without any arguments) should report that the _default_ loadparm is _NOW_ currently set to "FOO".

Is that correct?

If so, that seems perfectly reasonable to me!

Does anyone else have any objections to this change?

**Comment by arfineman** [comment]
Yes. This is exactly how I believe it should work.

**Comment by Fish-Git** [comment]
> Yes. This is exactly how I believe it should work.

Thought so. &nbsp; `:)`

Fix in progress...

**Comment by Fish-Git** [comment]
Fixed by commit 328e99a685b1a07367de14e5d7b80f449a9bc958.

Closing issue.
